### PR TITLE
Faster clearing of screen

### PR DESCRIPTION
**Pull request by sphereflow**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -91,10 +91,7 @@ where
         delay.delay_ms(200);
         self.write_command(Instruction::FRMCTR1, &[0x01, 0x2C, 0x2D])?;
         self.write_command(Instruction::FRMCTR2, &[0x01, 0x2C, 0x2D])?;
-        self.write_command(
```
Comment (sajattack) [inline src/lib.rs:94]: Why is this removed?

Comment (sajattack) [reply src/lib.rs:94]: Oh nevermind it was just rustfmt'ed into one line.

File: src/lib.rs [hunk 2]
```diff
@@ -314,6 +311,17 @@ where
 
         Ok(())
     }
+
+    fn clear(&mut self, color: Self::Color) -> Result<(), Self::Error> {
```
Comment (sajattack) [inline src/lib.rs:315]: Have you looked to see if there's any hardware clear in ST7735 that would be even faster than this?

Comment (sphereflow) [reply src/lib.rs:315]: After perusing the docs, as far as I can see, there is no single command to clear the entire display (or a portion of it) after sending a given color only once. It seems like the only way to change the contents of the screen while its running is to first set a rectangular section and then write color values pixel by pixel over the SPI bus.
